Search/Adapter/ElasticSearchAdapter.php

```diff
@@ -78,6 +78,11 @@ public function index(Document $document, $indexName)
                 case Field::TYPE_ARRAY:
                     $fields[$massiveField->getName()] = $value;
                     break;
+                case Field::TYPE_DATE:
```

Comment:
Probably the docs are only rebuild for master. And if I think about it, we should maybe always use converter. So there could be a default converter for string and array, then we could skip this part of the code. Or am I overlooking something? Anyway, this wouldn't be part of this PR, but if we decide to do so, the date could be implemented as converter already.